meta.yaml

```diff
@@ -1,6 +1,6 @@
 package:
   name: tomopy
-  version: '0.1.8'
+  version: '0.1.9'
 
 source:
   git_url: https://github.com/dgursoy/tomopy.git
```

Comment:
`git_tag`, `git_rev` and `git_branch` are all aliases of each other for now.

Another useful trick is to use a templated `meta.yaml` (see https://github.com/Nikea/scikit-xray/blob/master/conda-recipe/meta.yaml) which will set the version string based on the current commit and most-recent tag (ex v0.1.9.post7_g012a456 ) so that conda builds off of any point in master will sort correctly.